src/proxy/BaseProxy.ts

```diff
@@ -34,39 +33,17 @@ import {ClusterService} from '../invocation/ClusterService';
  */
 export abstract class BaseProxy {
 
-    protected readonly name: string;
-    protected readonly serviceName: string;
-    protected readonly proxyManager: ProxyManager;
-    protected readonly partitionService: PartitionService;
```

Comment:
If you are careful you can see that there are two unused property on this class. I deleted them